src/PermissionServiceProvider.php

```diff
@@ -124,30 +124,19 @@ public static function bladeMethodWrapper($method, $role, $guard = null): bool
 
     protected function registerBladeExtensions($bladeCompiler): void
     {
+        /** @var BladeCompiler $bladeCompiler */
```

Comment:
this part is not right, just add data type
```php
protected function registerBladeExtensions(BladeCompiler $bladeCompiler): void
{

//  OR add @param PHPDOC

/** @param BladeCompiler $bladeCompiler */
protected function registerBladeExtensions($bladeCompiler): void
{
```

The other looks good to me

Reply:
Yes, type-hinting the function parameter directly in the function declaration is the preferred way.

Reply:
Thanks for the suggestion; I've fixed it.